src/core/installAddon.js

```diff
@@ -548,18 +548,7 @@ export class WithInstallHelpers extends React.Component<WithInstallHelpersIntern
   }
 }
 
-type withInstallHelpersParams = {|
-  defaultInstallSource: string | null,
-|};
-
-export function withInstallHelpers({
-  defaultInstallSource,
-}: withInstallHelpersParams) {
-  invariant(
-    typeof defaultInstallSource !== 'undefined',
-    'defaultInstallSource is required',
-  );
-
+export function withInstallHelpers() {
```

Comment:
You can probably export the function directly, not a function that returns a function. That would be simpler and that would remove the need for the parentheses in `compose()`.

Reply:
```js
const InstallButtonWrapper: React.ComponentType<Props> = compose(
  withRouter,
  withInstallHelpers, // instead of withInstallHelpers()
  connect(mapStateToProps),
)(InstallButtonWrapperBase);
```

Reply:
`withInstallHelpers` with a lower case `w` is the function that creates the HOC instance. It is an instance of the HOC class `WithInstallHelpers` with a `W`.

You can probably write:

```js
export const withInstallHelpers = (WrappedComponent: React.ComponentType<any>) => {
```

Reply:
That works. :-)